subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -272,10 +258,23 @@ static int handle_device_control_update(const struct nct_evt *const evt,
 
 	*control_found = (status != NRF_CLOUD_CTRL_NOT_PRESENT);
 	if (*control_found) {
+
 #if IS_ENABLED(CONFIG_NRF_CLOUD_ALERTS)
 		nrf_cloud_alert_control_set(ctrl_data.alerts_enabled);
 #endif /* CONFIG_NRF_CLOUD_ALERTS */
+
 		nrf_cloud_log_control_set(ctrl_data.log_level);
+	} else {
+#if IS_ENABLED(CONFIG_NRF_CLOUD_ALERTS)
```

Comment:
nrf_cloud_alerts.c is not compiled in unless CONFIG_NRF_CLOUD_ALERTS is enabled, so it needs to be done with the preprocessor

Reply:
Ah, yeah -- I guess we could replace functions like this with no-op macros when the .c files aren't compiled, so that they can be referenced in external code without using the preprocessor

But switching to that is not for this PR